rust/cubesql/DEVELOPMENT.md

```diff
@@ -0,0 +1,17 @@
+# Running in standalone mode
+
+Prerequisites:
+
+- `rustup`
+
+```bash
+rustup default nightly-2021-07-04
```

Comment:
We should drop this line. After we splited Cube Store & Cube SQL into different workspaces it becomes useless because @cristipp put it a specific version

https://github.com/cube-js/cube.js/commit/a919f61e67c18864d87619021be087332c139ebe#diff-91f30fe149dd8b1116eeabe80d230ce30b4f44dfe070b4cbe99fa138835c796d